app/src/main/java/io/appium/uiautomator2/handler/GetElementAttribute.java

```diff
@@ -106,16 +107,33 @@ private static int getScrollableOffset(AndroidElement uiScrollable) {
         int y1 = bounds.centerY() + MINI_SWIPE_PIXELS;
         int x2 = x1;
         int y2 = y1 - (MINI_SWIPE_PIXELS * 2);
+
+        // ensure that our xs and ys are within the bounds of the element
+        if (y1 > bounds.width()) {
+            y1 = bounds.width();
```

Comment:
It's better to [avoid edges of element](https://android.googlesource.com/platform/frameworks/testing/+/master/uiautomator/library/core-src/com/android/uiautomator/core/UiScrollable.java#635).